CDP4SiteDirectory/Views/PersonBrowser/PersonBrowser.xaml

```diff
@@ -125,6 +125,7 @@
                 <dxg:TreeListColumn FieldName="GivenName" Header="First Name" SortIndex="0" />
                 <dxg:TreeListColumn FieldName="Surname" Header="Last Name" />
                 <dxg:TreeListColumn FieldName="Organization.ShortName" Header="Organization" />
+                <dxg:TreeListColumn FieldName="DefaultDomain.Name" Header="Default Domain"></dxg:TreeListColumn>
```

Comment:
what happens when the default domain is null?

Reply:
Empty field
Like here
![image](https://user-images.githubusercontent.com/104059834/167087708-27471679-1683-4e1f-8617-26a55935cd9e.png)